app/src/main/java/de/tum/in/tumcampusapp/database/repository/LocalRepository.kt

```diff
@@ -0,0 +1,11 @@
+package de.tum.`in`.tumcampusapp.database.repository
+
+import de.tum.`in`.tumcampusapp.models.cafeteria.Cafeteria
+import io.reactivex.Flowable
+
+
+interface LocalRepository {
```

Comment:
This should probably be named somewhat better: for example `CafeteriaRepository` ?

Reply:
depends whether or not we want to use a central repository or a repo for each model. I tend to have a central repo.

Reply:
A repo for each model. Check like the second heading: http://deviq.com/repository-pattern/